src/anna/week_1/OddOrEven.java

```diff
@@ -23,6 +26,9 @@ public static void oddOrEven(int n) {
         } else {
             System.out.println(n + " is odd number");
         }
+    }
 
+    public static String returnTypeOddOrEven(int num) {
```

Comment:
method returnTypeOddOrEven() looks great!

Reply:
Thank you